build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -136,172 +139,286 @@
     </GitCommitTime>
     <Touch Files="$(LlvmSourceFullPath)\Makefile.config.in" Time="$(_LlvmCommitTime)" />
   </Target>
+
   <Target Name="_PrepareLlvmItems">
     <ItemGroup>
-      <_LlvmSourceFile Include="$(LlvmSourceFullPath)\lib\**\*.cpp" />
-      <_LlvmHeaderFile Include="$(LlvmSourceFullPath)\lib\**\*.h" />
-      <_LlvmMakefileConfig Include="@(_LlvmRuntime->'$(_LlvmOutputDirTop)\%(BuildDir)\Makefile.config')" />
-      <_LlvmConfigureStamp Include="@(_LlvmRuntime->'$(_LlvmOutputDirTop)\%(BuildDir)\.stamp-configure')" />
+      <_LlvmSourceFileWithoutMonoSdks Include="$(LlvmSourceFullPath)\lib\**\*.cpp" Condition=" '%(_LlvmRuntime.UseMonoSdks)' == 'False' " />
```

Comment:
This `Condition` seems...wonky. _Normally_ a `%(Foo.Bar)` condition only makes sense if you're otherwise manipulating `@(Foo)`, e.g. the original:

```xml
<_LlvmSourceBinary Include="@(_LlvmRuntime->'$(_LlvmOutputDirTop)\%(BuildDir)\Release\bin\opt%(ExeSuffix)')" Condition=" '%(_LlvmRuntime.InstallBinaries)' == 'true' " /
```

The `Include` is using `@(_LlvmRuntime)`, so *within* the `@(_LlvmRuntime->'...')` expression, there is a `%(_LlvmRuntime.InstallBinaries)` which is "in scope."

*Here*, though, there is no `%(_LlvmRuntime.UseMonoSdks)` which is "in scope". My guess, then, is that this Condition is always evaluated as False.

Reply:
Yes, looks like a mistake on my end.